.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+## Exercism v2
```

Comment:
This file is on the master branch, but not on the nextercism branch.

I fetched your repo and checked out the branch; sure enough it has a lot of testVersion references, which have been removed from nextercism branch.  That says your run-length-encoding branch isn't a fresh rebase on nextercism.